frontend/src/components/filters/getFilterProps.js

```diff
@@ -48,31 +48,52 @@ function getInputFilterProps(column, descriptions, filters, setFilter, setFilter
       confirm()
   }
 
-  const onToggleSwitch = checked => {
+  const onToggleSwitch = (checked, option )=> {
+    setFilterOption(dataIndex, option, checked)
+  }
+
+  const onChangeRecursive = checked => {
+    onToggleSwitch(checked, 'recursiveMatch')
     setFilterOption(dataIndex, 'exactMatch', checked)
```

Comment:
Is it normal that `onChangeRecursive` sets `exactMatch` to the same value?

Reply:
Yes. You cant do recursive search with a non-exact value, so it will change automatically

Reply:
And what if the user unchecks `exactMatch` after having checked `recursiveMatch`? I would not fix the issue this way, I would make sure that `exactMatch` is ignored in the query params creation function, and maybe disabled `exactMatch` if `recursiveMatch` is checked.

Reply:
yes it is disabled